package-lock.json

```diff
@@ -13249,6 +13249,18 @@
                 "@microsoft/1ds-post-js": "^3.2.3"
             }
         },
+        "@vscode/test-electron": {
```

Comment:
Weird, it seems like the `package-lock.json` updated but I don't see these changes reflected in the `package.json` itself.

Reply:
Fixed. I slipped up when reverting the package.json changes that the new test package automatically makes (it annoyingly changes all the whitespace, etc).